scripts/setup

```diff
@@ -15,7 +15,7 @@ pip install -r requirements.txt --upgrade
 pip install -r requirements-rpc.txt --upgrade
 pip install -r requirements-dev.txt --upgrade
 # For python2
-if python -c 'import sys ; exit(sys.version_info < (3, 0))'
+if python -c 'import sys ; exit(not sys.version_info < (3, 0))'
```

Comment:
This was not working before?

Reply:
Yes, it was not working before.

On my system, `python -c 'import sys ; exit(True)'; echo "exit(True) in python results in an exit code of $?"; python -c 'import sys ; exit(False)'; echo "exit(False) in python results in an exit code of $?"` prints:

```
exit(True) in python results in an exit code of 1
exit(False) in python results in an exit code of 0
```

However, bash interprets an exit code of 0 as true/success for flow control purposes, so it ends up running `pip install -r requirements-dev2.txt --upgrade` in a python 3 virtualenv, which broke a couple of tests.